fe/fe-core/src/main/java/com/starrocks/warehouse/Warehouse.java

```diff
@@ -53,4 +54,8 @@ public void write(DataOutput out) throws IOException {
         String json = GsonUtils.GSON.toJson(this);
         Text.writeString(out, json);
     }
+
+    public long getWorkerGroupId() {
```

Comment:
this interface is not right. There are clusters under warehouse, the cluster is a 1:1 mapping to worker group id.

Reply:
You should be aware of why we need a conversion from warehouse to work group. So, are you suggesting we modify the method name? To reflect the concept that warehouse and workGroup have a one-to-many relationship, such as getAnyWorkerGroupId?

Reply:
getAny... or `List<Long> getWorkerGroupIds()`, and in current implementation, there is only one element in the list.